scripts/report_errors.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+export ERROR_LOG="/tmp/error.log"
+ERROR_MSG=$(cat $ERROR_LOG)
+
+function report_error_to_github {
+    if [ $? -eq 0 ]; then
+        exit 0
+    fi
+
+    echo $ERROR_MSG
+
+    if [ -z $CIRCLE_PR_NUMBER ]; then
```

Comment:
I think this also needs a check to see whether it is running inside circle at all.